spk/go2rtc/Makefile

```diff
@@ -10,7 +10,7 @@ UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 MAINTAINER = skrashevich
 DESCRIPTION = Ultimate camera streaming application with support RTSP, RTMP, HTTP-FLV, WebRTC, MSE, HLS, MP4, MJPEG, HomeKit, FFmpeg, etc
 DISPLAY_NAME = go2rtc
-CHANGELOG = "Enhanced Rockchip transcoding, OPUS codec support, WebUI log viewer, Tapo and HomeKit source handling. Improved H265, FFmpeg, CORS support. Fixed D-Link SDP, exec source issues, H265 hardware profiles, OPUS audio, mdns log, Aqara G2H compatibility, reconnect panic."
+CHANGELOG = "The latest update includes various improvements and bug fixes such as enhanced ASCII streaming support, WebRTC loading optimizations, and fixes for HomeKit configuration issues."
```

Comment:
We are used to document changes related to the synology package only, and not related to the upsteam application.
If you want, you can provide link(s) to upstream change log(s) (see spk/bazarr/Makefile for an example).